src/enums/move-use-mode.ts

```diff
@@ -79,7 +80,8 @@ export type MoveUseMode = ObjectValues<typeof MoveUseMode>;
 /**
  * Check if a given {@linkcode MoveUseMode} is virtual (i.e. called by another move or effect).
```

Comment:
```suggestion
 * Check if a given `MoveUseMode` is virtual (i.e. called by another move or effect).
```
You could remove it from the `@param` section instead, but I like having that be where the links are.